---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

For the up-to-date publication list, see my Google Scholar profile <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>. Most of my publications are freely available on PubMed Central. I will post links to PMC and post pdfs. However some recent papers may not be avaialble, please let me know, you are unable to access a manuscript.


In Reverse Chronological order

1.	**Culhane AC**, Hall MD, Rothwell NJ, Luheshi GN. Cloning of interleukin-18 from rat brain cDNA. Mol. Psychiatry  1998; 3: 362-366 
2.	Wheeler RD, **Culhane AC**, Hall MD, Pickering-Brown S, Rothwell NJ, Luheshi GN. Detection of the interleukin-18 family in the rat brain by RT-PCR. Mol. Brain Res. 2000; 77: 290-293 
3.	**Culhane AC**, Perriére G, Considine EC, Cotter TG, Higgins DG. Between-group analysis of microarray data. Bioinformatics .2002; 18: 1600-1608  
   (Recommended by Faculty of 1000. http://www.f1000biology.com/article/12490444/)
4.	Culhane AC*, Perriére G, Higgins DG. Cross-platform comparison and visualisation of gene expression data using co-inertia analysis. BMC Bioinformatics 2003; 4(1):59 
5.	McArdle L, McDermott M, Purcell R, Grehan D, O'Meara A, Breatnach F, Catchpoole D, **Culhane AC**, Jeffery I, Gallagher WM, Stallings RL. Oligonucleotide microarray analysis of gene expression in neuroblastoma displaying loss of chromosome 11q. Carcinogenesis 2004; 25:1599-1609 
6.	Kapushesky M, Kemmeren P, **Culhane AC**, Durinck S, Ihmels J, Körner C, Kull M, Torrente A, Sarkans U, Vilo J, Brazma A. Expression Profiler: next generation: an online platform for analysis of microarray data. Nucleic Acids Research. 2004; 32: W465-470
7.	Gallagher WM, Orla E. Bergin OE, Rafferty M, Kelly ZD, Nolan IM, Fox EJP, **Culhane AC**, McArdle L, Fraga MF, Hughes L, Currid CA, O’Mahony F, Byrne A, Murphy AA, Moss C, McDonnell S, Stallings RL, Plumb JA, Esteller M, Brown R, Dervan PA, Easty DJ  Multiple markers for melanoma progression regulated by DNA methylation: Insights from transcriptomic studies Carcinogenesis. 2005; 26(11):1856-1867. 
8.	Baty F, Bihl MP , Perriére G, **Culhane AC**, Brutsche MH. Optimization of the Between Group Analysis for disease class prediction based on gene expression data: a new jackknife based gene selection procedure. BMC Bioinformatics. 2005; 6: 239 
9.	**Culhane AC**, Thioulouse J, Perriére G, Higgins DG. MADE4: An R package for multivariate analysis of gene expression data. Bioinformatics.2005; 21(11):2789-90 
10.	Jeffery IB, Higgins DG, Culhane AC*. Comparison and evaluation of microarray feature selection methods. BMC Bioinformatics 2006; 26;7:359.   
(BMC Online Highly Accessed Manuscript)
11.	Culhane AC*, Thioulouse J A multivariate approach to integrating datasets using made4 and ade4. R News. Special Issue on Bioconductor 2006; 6(5) 54:58.
12.	Jeffery IB,Madden SF, McGettigan PA, Perriere G, **Culhane AC**, Higgins DG. Integrating transcription factor binding site information with gene expression datasets. Bioinformatics 2007; 23(3) 298-305
13.	O’Brien SL, Fagan A, Fox EJP, Millikan RC, **Culhane AC**, Brennan DJ, McCann AH, Hegarty S, Moyna S, Duffy MJ, Higgins DG, Jirstrom K, Landberg G, Gallagher WM.. CENP-F expression is associated with poor prognosis and chromosomal instability in patients with primary breast cancer. Int. J. Cancer 2007; 120(7):1426-35
14.	Sobhian B, Shao G, **Culhane AC**, Moreau L, Xia B, Livingston DM, Greenberg RA. Rap80 targets BRCA1 to specific ubiquitin structures at DNA damage sites. Science. 2007; 316(5828): 1198-202. 
(Recommended by Faculty of 1000. http://www.f1000biology.com )
15.	Fagan A, **Culhane AC**, Higgins DG. A multivariate analysis approach to the integration of proteomic and gene expression data. Proteomics. 2007; 7(13):2162-71. (Co-Corresponding) 
16.	Chittenden TW, Howe EA, **Culhane AC**, Sultana R, Taylor JM, Holmes C, Quackenbush J. Functional classification analysis of somatically mutated genes in human breast and colorectal cancers. Genomics.  2008 91(6):508-11
17.	Brennan DJ, Rexhepaj E, O’Brien SL, Mc Sherry E, O’ Connor DP, Fagan A, Culhane AC, Higgins DG, Jirstrom KC, Millikan RC, Landberg G, Duffy MJ, Hewitt SM, Gallagher WM. Altered Nuclear-Cytoplasmic Ratio of Survivin is a Prognostic Indicator in Breast Cancer. Clinical Cancer Research. 2008; 14(9):2681-9 
18.	Smith MJ, **Culhane AC**, Killeen S, Kelly MA, Wang JH, Cotter TG, Redmond HP. Mechanisms driving local breast cancer recurrence in a model of breast conserving surgery.  Annals of Surgical Oncology 2008; 15(10):2954-64
19.	Cheng ASL, **Culhane AC**, Chan MWY, Chinnambally V, Ehrich M, Nasir A, Rodriguez BAT, Liu J, Yan PS, Quackenbush J, Nephew KP, Yeatman TJ, and Huang TH-M.  Epithelial Progeny of Estrogen-exposed Breast Progenitor Cells Display a Cancer-like Methylome. Cancer Research. 2008; 68(6): 1786-96.  
20.	Smith MJ, **Culhane AC**, Donovan M, Kelly MA, Coffey C, Barry B, Wang JH, Kirwan W, Higgins DG, Cotter TG, Redmond HP.  Analysis of Differential Gene Expression in Colorectal Cancer and Stroma Using Flow Activated Cell Sorting Purification.  British Journal of Cancer. 2009; 100(9):1452-64.
21.	Ioannidis JP, Allison DB, Ball CA, Coulibaly I, Cui X, **Culhane AC**, Falchi M, Furlanello C, Game L, Jurman G, Mangion J, Mehta T, Nitzberg M, Page GP, Petretto E, van Noort V.  Repeatability of published microarray gene expression analyses.  Nature Genetics. 2009; 41(2):149-155. 
22.	Speers C, Tsimelzon A, Sexton K, Herrick AM, Gutierrez C, **Culhane A**, Quackenbush J, Hilsenbeck S, Chang J, Brown P.  Identification of Novel Kinase Targets for the Treatment of Estrogen Receptor-Negative Breast Cancer. Clinical Cancer Research; 2009. 15(20):6327-40.
23.	**Culhane AC**, Quackenbush J. Confounding Effects in "A Six-gene signature Predicting Breast Cancer Lung Metastasis" Cancer Research. 2009; 69(18): 7480-5.
24.	Chen D-T, Nasir A, **Culhane AC**, Venkataramu C, Fulp W, Rubio R, Wang R, Agrawal D, Gruidl M, Bloom G, Anderson T, White J, Quackenbush J, Yeatman T. Proliferative genes dominate malignancy-risk gene signature in histologically-normal breast tissue Breast Cancer Res Treat. 2010; 19(2): 335-346.
25.	**Culhane AC**, Schwarzl T, Sultana R, Picard KC, Picard SC, Lu TH, Franklin K, French SJ, Papenhausen J, Correll M, Quackenbush J. GeneSigDB - a curated database of gene expression signatures. Nucleic Acids Research (database issue). 2010 Database Issue: D716-25 
26.	Chittenden TW, Pak J, Rubio R, Cheng H, Holton K, Prendergast N, Glinskii V, Cai Y, **Culhane AC**, Bentink S, Schwede M, Mar J, Howe EA, Aryee M, Sultana R, Lanahan AA, Taylor JM, Holmes C, Hahn WC, Zhao JJ, Iglehart, JD, Quackenbush J. Therapeutic Implications of GIPC1 Silencing in Cancer. PLoS ONE. 2010; 5(12)
27.	Konstantinopoulos PA, Cannistra SA, Fountzilas H, **Culhane A**, Pillay K, Rueda B, Cramer D, Seiden M, Birrer M, Coukos G, Zhang L, Quackenbush J, Spentzos D. (2011) Integrated analysis of multiple microarray datasets identifies a reproducible survival predictor in ovarian cancer. PLoS One.6(3):e18202.
28.	Schröder M, **Culhane AC**, Quackenbush J, Haibe-Kains B (2011) SurvComp: an R/Bioconductor package for performance assessment and comparison for survival analysis. Bioinformatics 27(22):3206-8 Supplement
29.	Gusenleitner D, Howe EA, Bentink S, Quackenbush J, **Culhane AC**. iBBiG: Iterative Binary Bi-clustering of Gene Sets. Bioinformatics. 2012. 28(19):2484-92. 
30.	Haibe-Kains B, Desmedt C, Loi S, **Culhane AC**, Bontempi G, Quackenbush J, Sotiriou C. A three-gene model to robustly identify breast cancer molecular subtypes. J Natl Cancer Inst. 2012;104(4):311-25. 
31.	**Culhane AC**, Schröder MS, Sultana R, Picard SC, Martinelli EN, Kelly C, Haibe-Kains B, Kapushesky M, St Pierre AA, Flahive W, Picard KC, Gusenleitner D, Papenhausen G, O'Connor N, Correll M, Quackenbush J. GeneSigDB: a manually curated database and resource for analysis of gene expression signatures. Nucleic Acids Res. 2012;40:D1060-6 
32.	Wang ZC , Birkbak NJ, **Culhane AC**, Drapkin RI, Fatima A, Tian R, Schwede M, Alsop K, Daniels KE, Piao H, Liu J, Etemadmoghadam D, Miron A, Salvesen HB, Mitchell G, DeFazio A, Quackenbush J, Berkowitz RS, Bowtell DD, Matulonis UA. Profiles of genomic instability in high-grade serous ovarian cancer predict treatment outcome. Clinical Cancer Research. 2012, 18(20):5806-15
33.	Bentink S, Haibe-Kains B, Risch T, Fan JB, Hirsch MS, Holton K, Rubio R, April C, Chen J, Wickham-Garcia E, Liu J, **Culhane A**, Drapkin R, Quackenbush J, Matulonis UA. Angiogenic mRNA and microRNA gene expression signature predicts a novel subtype of serous ovarian cancer. PLoS One. 2012;7(2):e30269. 
34.	Ho Sui SJ, Begley K, Reilly D, Chapman B, McGovern R, Rocca-Sera P, Maguire E, Altschuler GM, Hansen TA, Sompallae R, Krivtsov A, Shivdasani RA, Armstrong SA, **Culhane AC**, Correll M, Sansone SA, Hofmann O, Hide W. The Stem Cell Discover Engine: an integrated repository and analysis system for cancer stem cell comparisons. Nucleic Acids Res. 2012;40 (Database issue):D984-91 
35.	Kapushesky M, Adamusiak T, Burdett T, **Culhane A**, Farne A, Filippov A, Holloway E, Klebanov A, Kryvych N, Kurbatova N, Kurnosov P, Malone J, Melnichuk O, Petryszak R, Pultsin N, Rustici G, Tikhonov A, Travillian RS, Williams E, Zorin A, Parkinson H, Brazma A. Gene Expression Atlas update--a value-added database of microarray and sequencing-based functional genomics experiments. Nucleic Acids Res. 2012;40:D1077-81. 
36.	Brennan DJ, O'Connor DP, Laursen H, McGee SF, McCarthy S, Zagozdzon R, Rexhepaj E, **Culhane AC**, Martin FM, Duffy MJ, Landberg G, Ryden L, Hewitt SM, Kuhar MJ, Bernards R, Millikan RC, Crown JP, Jirström K, Gallagher WM. The cocaine- and amphetamine-regulated transcript mediates ligand-independent activation of ERα, and is an independent prognostic factor in node-negative breast cancer. Oncogene. 2012;31(30):3483-94. 
37.	Bowman-Colin C, Xia B, Bunting S, Klijn C, Drost R, Bouwman P, Fineman L, Chen X, **Culhane AC**, Cai H, Rodig SJ, Bronson RT, Jonkers J, Nussenzweig A, Kanellopoulou C, Livingston DM. Palb2 synergizes with Trp53 to suppress mammary tumor formation in a model of inherited breast cancer. Proc Natl Acad Sci U S A.  2013. 110(21):8632-7.
38.	Schröder M, Gusenleitner D, Quackenbush J, **Culhane AC**, Haibe-Kains B. RamiGO: an R/Bioconductor package providing an AmiGO Visualize interface. Bioinformatics 2013 29(5):666-8 (joint last author)
39.	Schwede M, Spentzos D, Bentink S, Hofmann O, Haibe-Kains B, Harrington D, Quackenbush J, **Culhane AC**. Stem cell-like gene expression in ovarian cancer is prognostic and associated with Type II tumor characteristics.  PLoS e. 2013 8(3):e57799
40.	Beck AH, Knoblauch NM, Hefti MH, Kaplan J, Schnitt SJ, **Culhane AC**, Schroeder M, Risch T, Quackenbush J, Haibe-Kains B. Significance Analysis of Prognostic Signatures. PLoS Computational Biology. 2013. 9 (1): e1002875
41.	Prencipe M, Madden SF. O’Neill A, O’Hurely G, **Culhane AC**, O’Connor D, Klocker H, Gallagher W, Watson W. Identification of transcription factors associated with castration-resistance: is the Serum Responsive Factor a potential therapeutic target?  Prostate. 2013. 73(7):743-53
42.	Kelly AD, Haibe-Kains B, Janeway KA, Howe EA, Goldsmith J, Kurek K, Perez-Atayde AR, Francoeur N, Fan JB,  Craig A, Schneider H, Gebhardt MC, **Culhane A**, Quackenbush J, Spentzos D. MicroRNA and mRNA paraffin-based studies in osteosarcoma reveal reproducible independent prognostic profiles at 14q32 Genome Medicine. 2013. 5(1):2
43.	Waldron L, Haibe-Kains B, **Culhane AC**, Riester M, Ding J, Wang V, Tyekucheva S, Bernau C, Risch T, Ganzfried B, Huttenhower C, Birrer M, Parmigiani G.  Comparative meta-analysis of prognostic gene signatures for late-stage ovarian cancer. J Natl Cancer Inst. 2014. 106(5).
44.	Riester M, Wei W, Waldron L, **Culhane AC**, Trippa L, Oliva E, Kim SH, Michor F, Huttenhower C, Parmigiani G, Birrer MJ. Risk Prediction for Late-Stage Ovarian Cancer by Meta-analysis of 1525 Patient Samples. J Natl Cancer Inst. 2014. 106(5)
45.	Santagata S, Thakkar A, Ergonul A, Wang B, Woo T, Hu R, Harrell JC, McNamara G, Schwede M, **Culhane AC**, Kindelberger D, Rodig S,  Richardson A, Schnitt SJ, Tamimi R, Ince TA. Taxonomy of Breast Cancers Based on Normal Cell Phenotype and Ontology. J Clin Invest. 2014 124(2):859-70.
46.	Meng C, Kuster B, **Culhane AC**, Gholami AM, A multivariate approach to the integration of multi-omics datasets.  BMC Bioinformatics 2014. 15(1):162.
47.	Kochupurakkal BS, Wang ZC, Hua T, Culhane AC, Rodig SJ, Rajkovic-Molek K, Lazaro JB, Richardson AL, Biswas DK, Iglehart JD. RelA-Induced Interferon Response Negatively Regulates Proliferation. PLoS One. 2015 Oct 13;10(10):e0140243. doi: 10.1371/journal.pone.0140243  (Co-corresponding author(
48.	Yu L, Li C, Yan Y, Shi Z, Picard S, Martinelli E, Martinelli V, Huang Z, Yang Y, Sultana R, Quackenbush J. Cheng L, **Culhane AC.**  GeneSigCatcher: an automated pipeline for the retrieval of most relevant PubMed Central articles for GeneSigDB.  Database. 
49.	Neupane M, Clark AP, Landini S, Birkbak NJ, Eklund AC, Lim E, **Culhane AC**, Barry WT, Schumacher SE, Beroukhim R, Szallasi Z, Vidal M, Hill DE, Silver DP. MECP2 Is a Frequently Amplified Oncogene with a Novel Epigenetic Mechanism That Mimics the Role of Activated RAS in Malignancy. Cancer Discov. 2016 Jan;6(1):45-58.
50.	Zeleznik OA, Thallinger GG, Platig J, **Culhane AC** Topological pathway enrichment analysis of gene expression in high grade serous ovarian cancer reveals tumor-stoma cross-talk. In Ainsbury E., Calle M., Cardis E., Einbeck J., Gómez G., Puig P (Eds). Trends in Mathematics, 2017. 7: 59-63. Birkhäuser, Cham
51.	deVelasco G, **Culhane AC**, Fay AP, Hakimi AA, Voss MH, Tannir NM, Tamboli P, Appleman LJ, Bellmunt  J, Rathmell WK, Albiges  L, Hsieh JJ, Heng DYC, Signoretti S, Choueiri TK.  Prognostic value of genomic signatures in metastatic Clear Cell Renal Cell Carcinoma (mRCC) using The Cancer Genome Atlas (TCGA) data.  The Oncologist, 2017. 22(3):286-292 (Co-first author)
52.	Ramos M, Schiffer L, Re A, Azhar R, Basunia A, Cabrera CR, Chan T, Chapman P, Davis S, Gomez-Cabrero D, **Culhane AC**, Haibe-Kains B, Hansen KD, Kodali H, Louis MS, Mer A, Riester M, Morgan M, Carey V. Software for the integration of multi-omics experiments in Bioconductor. Cancer Research, 2017 77(21):e39-e42. doi: 10.1158/0008-5472.CAN-17-0344
53.	Thorsson V, Gibbs DL, Brown SD, Wolf, D, Bortone DS, Ou Yang TH Gao G, Plaisier CL, Eddy J, Ziv E, **Culhane AC**, et al., The Immune Landscape of Cancer. Immunity. 2018 48 (4):812-830 
54.	Li AG, Murphy EC, **Culhane AC**, Powell E, Wang H, Bronson RT, Von T,Giobbie-Hurder A, Gelman RS, Briggs KJ, Piwnica-Worms H, Zhao JJ, Kung AL, Kaelin, WG Jr, Livingston DM. BRCA1-IRIS promotes human tumor progression through PTEN blockade and HIF-1α activation. Proc Natl Acad Sci U S A. 2018 2: 9;115(41):E9600-E9609. doi: 10.1073/pnas.1807112115
55.	Korkut A, Zaidi S, Kanchi RS, Rao S, Gough NR, Schultz A, Li X, Lorenzi PL, Cancer Genome Atlas Research Network, et al., A Pan-Cancer Analysis Reveals High-Frequency Genetic Alterations in Mediators of Signaling by the TGF-β Superfamily. Cell Syst. 2018 Sep 14. pii: S2405-4712(18)30357-0. doi: 10.1016/j.cels.2018.08.010 . 
56.	Meng C, Basunia A, Peters B, Gholami AM, Kuster B, **Culhane AC**. MOGSA: Integrative Single Sample Gene-set Analysis of Multiple Omics Data. Mol Cell Proteomics. 2019 18(8 suppl 1): S153-S168. doi: 10.1074/mcp.TIR118.001251. PubMed Central PMCID: PMC6692785.
57.	Schwede M., Waldron L., Mok S.C., Wei W., Basunia A., Merritt M.A, Parmigiani G., Harrington D.P., Quackenbush J., Birrer  M.J., **Culhane A.C.** The impact of stroma on the discovery of molecular subtypes and prognostic gene signatures in serous ovarian cancer.  Cancer Epidemiology, Biomarkers & Prevention. 2020 1055-9965.EPI-18-1359
58.	Hsu LL, **Culhane AC**. Impact of Data Preprocessing on Integrative Matrix Factorization of Single Cell Data. Front. Oncol. 2020. 10 DOI: 10.3389/fonc.2020.00973 . 
(Top 20% of articles in Frontiers. 826 downloads, 6,439 Views. Altmetric 110 -top 5%))
59.	Lane JCE, Weaver J, Kostka K, Duarte-Salles T, Abrahao MTF, Alghoul H, Alser O, Alshammari TM, Biedermann P, Burn E, Casajust P, Conover M, **Culhane AC**, Davydov A et al., Safety of hydroxychloroquine, alone and in combination with azithromycin, in light of rapid wide-spread use for COVID-19: a multinational, network cohort and self-controlled case series study. Lancet Rheumatol. 2020 Nov;2(11):e698-e711. doi: 10.1016/S2665-9913(20)30276-9. 
60.	Burn E, You SC, Sena AG, Kostka K, Abedtash H, Abrahão MTF, Alberga A, Alghoul H, Alser O, Alshammari TM, Aragon M, Areia C, Banda JM, Cho J, **Culhane AC** et al. Deep phenotyping of 34,128 patients hospitalised with COVID-19 and a comparison with 81,596 influenza patients in America, Europe and Asia: an international network study. Nature Comm. 2020. Oct 6;11(1):5009. doi: 10.1038/s41467-020-18849-z.PMID: 33024121 
61.	Dhimolea E, de Matos Simoes R, Kansara D, Weng X, Sharma S, Awate P, Liu Z, Gao D, Mitsiades N, Schwab J, Chen Y, Jeselsohn R, **Culhane AC**, Brown M, Georgakoudi I, Mitsiades CS. Endocrine resistance through pleiotropic mechanisms in the 3-dimensional bone microenvironment. Cancer Research. 2020 81 (2), 371-383 https://doi.org/10.1158/0008-5472.CAN-20-0571
62.	Dhimolea E, De Matos Simoes R, Kansara D, Weng X, Sharma S, Bouyssou J, Raja J, Awate P, Tang H, Glassner B, Liu Z, DeRose Y, Gao G, Bryan J, Wang L, Bender S, Roth J, Scheffer M, Jeselsohn R, Gray N, Georgakoudi I, Vazquez F, Tsherniak A, Steidl U, Chen Y, Welm A, Bartholdy B, Brown M, **Culhane AC**, Mitsiades C. An embryonic diapause-like state with suppressed Myc activity enables tumor treatment persistence Cancer Cell  2021. 39 (2), 240-256. e11
63.	Shirasaki R, Matthews GM, Gandolfi1 S, de Matos Simoes R, Buckley DL, Raja J, Sievers QL, Brüggenthies JB, Dashevsky O, Poarch H, Tang H, Bariteau MA, Sheffer M, Hu Y, Downey-Kopyscinski SL, Hengeveld PJ, Glassner BJ, Dhimolea E, Ott CJ, Zhang T, Kwiatkowski NP, Laubach JP, Schlossman RL, Richardson PG, **Culhane AC**, Groen RMJ, Fischer ES, Vazquez F, Tsherniak A, Hahn WC, Levy J, Auclair D, Licht JD, Keats J, Boise LH, Ebert BL, Bradner JE, Gray NS, Mitsiades CS.  Functional genomics identify distinct and overlapping genes mediating resistance to different classes of heterobifunctional degraders of oncoproteins. Cell Reports. 34, 108532
64.	Sheffer M, Lowry M, Beelen N,  Borah M, Naffar-Abu Amara S, Mader CC, Roth JA, Tsherniak A, Freeman SS, Dashevsky O, Gandolfi S, Bender S, Bryan JG, Zhu C, Wang L, Tariq I, Kamath GM, De-Matos Simoes R, Dhimolea E, Yu C, Hu Y, Dufva O, Giannakis M, Syrgkanis V, Fraenkel E,  Golub T, Romee R, Mustjoki S, **Culhane AC**, Wieten L, Mitsiades CS. Landscape of molecular events regulating tumor cell responses to natural killer cells Nature Genetics (In Press)
65.	Roel, E., Pistillo, A., Recalde, M., Sena, A.G., Fernández-Bertolín, S., Aragón, M., Puente, D., Ahmed, W.-U.-R., Alghoul, H., Alser, O., Alshammari, T.M., Areia, C., Blacketer, C., Carter, W., Casajust, P., **Culhane, A.C.**, et al., 2021. Characteristics and outcomes of 118,155 COVID-19 individuals with a history of cancer in the United States and Spain. [MedRxiv](https://www.medrxiv.org/content/10.1101/2021.01.12.21249672v1).

Other peer reviewed publications (Reviews, letters to the editor)
===========
66.	Hale AJ, Smith CA, Sutherland LC, Stoneman VE, Longthorne V, **Culhane AC**, Williams GT. Apoptosis: molecular regulation of cell death. Eur. J. Biochem. 1996; 236: 1-26
67.	Duffy MJ, Kelly ZD, **Culhane AC**, O’ Brien S, Gallagher WM. DNA Microarray-Based Gene Expression profiling in cancer: Aiding cancer diagnosis, assessing prognosis and predicting response to therapy. Current Pharmacogenomics. 2005 16: 289-304 
68.	Brennan D, O’ Brien S, Fagan A, **Culhane AC**, Higgins DG, Duffy MJ, Gallagher WM. Application of DNA microarray technology in determining breast cancer prognosis and therapeutic response. Expert Opinion on Biological Therapy. 2005; 5(8):1069-83 
69.	**Culhane AC** and Howlin J. Omic Approaches to Understanding Breast Cancer. Cellular and Molecular Life Sciences. 2007.
70.	Meng C, Zeleznik OA, Thallinger GG, Kuster B, Gholami AM, **Culhane AC**, Dimension reduction techniques for the integrative analysis of multi-omics data. Special issue on Computational Systems Biomedicine in  Briefings in Bioinformatics 2016 Mar 11. pii: bbv108.
71.	Kannan L, Ramos M, Re A, El-Hachem N, Safikhani Z, Gendoo DM, Davis S, Gomez-Cabrero D, Castelo R, Hansen KD, Carey VJ, Morgan M, **Culhane AC**, Haibe-Kains B, Waldron L. Public data and open source tools for multi-assay genomic investigation of disease. Special issue on Computational Systems Biomedicine in Briefings in Bioinformatics. 2015 Oct 12. pii: bbv080
72.	Meng C & **Culhane AC** Integrative exploratory analysis of two or more genomic data. in Statistical Genomics: Methods and Protocols. Davis S (Editor). Methods Mol Biol. 2016; 1418:19-38. 
73.	de Velasco G, **Culhane AC**, Heng DYC, Signoretti S, Choueiri TK (2017) In Reply. The issue of tissue stratification. Oncologist. 2017 Sep 11. pii: theoncologist.2017-0283. doi:
74.	Stein-O'Brien GL,  Arora R,  **Culhane AC**,  Favorov A,  Garmire LX,  Greene C,  Goff LA,  Li Y,  Ngom A,  Ochs MF,  Xu Y,  Fertig EJ (2018) Enter the matrix: factorization uncovers knowledge from omics  Trends Genet. 2018 Oct;34(10):790-805. doi: 10.1016/j.tig.2018.07.003.

Book Chapters, Books and monographs
===========
75.	Brazma  A., and **Culhane AC**. Algorithms for gene expression analysis. In Encyclopedia of Genetics, Genomics, Proteomics and Bioinformatics. Jorde LB., Little PFR, Dunn MJ., Subramaniam S, editors. John Wiley & Sons. London. 2005, p 3148 -3159  (John Quackenbush, editor. Computational Methods for High-throughput Genetic Analysis: Expression Profiling; vol. 7(5)) 
76.	**Culhane AC** and Brazma A. Mining of gene expression data. In In Silico technology in drug target identification and validation.  León D, Markel S., editors.  Marcel Dekker, Inc. New York. CRC Press, Drug Discovery Series. 2006 p123-139
77.	Djebbari A, **Culhane AC**, Armstrong AJ, Quackenbush J. (2008)  Artificial intelligence methods for analyzing microarray data.  In Encyclopedia of Artificial Intelligence. Rabuñal, JR, Dorado J & Pazos A (Editors)
78.	De Troyer E, Sengupta E, Otava M, Zhang JD, Kaiser S, **Culhane AC**, Gusenleitner D, Gestraud P, Csardi G, Hochreiter S, Klambauer G,  Clevert DA, Perualila NJ, Kasim A,  Verbeke T and Shkedy Z. (2016) The biclustGUI Shiny App. Biclustering. In Applied Biclustering Methods for Big and High-Dimensional Data Using R Kasim A, Shkedy Z, Kaiser S, Hochreiter S, Talloen W (Editors) Chapman & Hall/CRC Press.
79.	De Troyer E, Otava M, Zhang JD, Pramana S, Khamiakova T, Kaiser S, **Culhane AC**, Gusenleitner D, Gestraud P, Csardi G, Aregay M, Hochreiter S, Klambauer G,  Clevert DA, Perualila NJ, Kasim A, Verbeke T and Shkedy Z. The BiclustGUI Package. In Applied Biclustering Methods for Big and High-Dimensional Data Using R Kasim A, Shkedy Z, Kaiser S, Hochreiter S, Talloen W (Editors) Chapman & Hall/CRC Press.


PrePrints 
===========
Hill KE, Rattani A, Lietz CE, Garbutt C, Choy E, Cote GM, **Culhane AC**, Kelly AD, Haibe-Kains B, Spentzos D. Machine learning-based prediction of response to PARP inhibition across cancer types. medRxiv preprint Sep. 27, 2019  http://dx.doi.org/10.1101/19007757

**Culhane AC** Carey VJ, Argelaguet R, Holmes S, Love MI, Yuan GC, Abadi AJ, Greene CS, Subramanian A, Singh A, Coullomb A, Pancaldi V, Arora A, Sodicoff JS, Welch JD, David-Marcisak EF, Ritchie M, You Y, Stein-O’Brien G, Sankaran K, Dries R, Deshpande A, Fertig E,  Le Cao KA Community-wide hackathons establish foundations for emerging single cell data integration. https://birsbiointegration.github.io/whitePaper/ 


